pkg/storage/docstore/docstore.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package docstore
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+)
+
+// Connection is a pairwise relation between two DIDs (ours and theirs)
+type Connection struct {
+	Mine   *did.Doc
+	Theirs *did.Doc
+	Label  string
+}
+
+// Provider storage provider interface
+type Provider interface {
+	GetStoreHandle(name string) (Store, error)
```

Comment:
Need godocs on both these methids

Reply:
Added godocs to the function definition.